openidm-ui/openidm-ui-enduser/pom.xml

```diff
@@ -1,164 +1,135 @@
-                  <type>zip</type>
-                  <classifier>www</classifier>
-                </artifactItem>
-
-                <artifactItem>
-                  <groupId>org.forgerock.openidm</groupId>
-                  <artifactId>openidm-ui-common</artifactId>
-                  <version>${project.version}</version>
-                  <type>zip</type>
-                  <classifier>www</classifier>
-                </artifactItem>
-              </artifactItems>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
-        <extensions>true</extensions>
-
-        <executions>
-          <execution>
-            <id>bundle-manifest</id>
-            <phase>package</phase>
-
-            <goals>
-              <goal>manifest</goal>
-            </goals>
-          </execution>
-        </executions>
+    Copyright 2011-2014 ForgeRock AS.
+    Portions Copyright 2017-2022 Wren Security.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-        <configuration>
-          <instructions>
-            <Import-Package>*</Import-Package>
-            <Include-Resource>
-              default=target/www
-            </Include-Resource>
-          </instructions>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+    <parent>
+        <groupId>org.forgerock.openidm</groupId>
+        <artifactId>openidm-ui</artifactId>
+        <version>6.0.3-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>openidm-ui-enduser</artifactId>
+    <!-- Bundle packaging retained because of common resource inclusion -->
+    <packaging>bundle</packaging>
+
+    <name>Wren:IDM - UI - End-user</name>
+    <description>Provides the UI components used by regular, non-admin users of a Wren:IDM deployment.</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.forgerock.commons.ui</groupId>
+            <artifactId>forgerock-ui-user</artifactId>
+            <version>${forgerock-ui.version}</version>
+            <type>zip</type>
+            <classifier>www</classifier>
+        </dependency>
+
+        <dependency>
+            <groupId>org.forgerock.openidm</groupId>
+            <artifactId>openidm-ui-common</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <classifier>www</classifier>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.rhino</artifactId>
```

Comment:
Is Rhino required? I know that it was there before the refactor, but I am wondering what it is being used for in this bundle.

Reply:
I'm not sure. I can create dedicated issue to check it.